pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go

```diff
@@ -53,31 +47,15 @@ func (r *MeshDefaultsReconciler) Reconcile(ctx context.Context, req kube_ctrl.Re
 		mesh.GetMeta().(*k8s.KubernetesMetaAdapter).Annotations = map[string]string{}
 	}
 	mesh.GetMeta().(*k8s.KubernetesMetaAdapter).GetAnnotations()[common_k8s.K8sMeshDefaultsGenerated] = "true"
-	r.Log.Info("marking mesh that default resources were generated", "mesh", req.Name)
```

Comment:
I find this log useful

Reply:
My thinking was you either get an error or you can see on the mesh object that it has the annotation so the message is sort of superfluous in terms of ensuring that the controller is functioning

Reply:
The difference is that with a log you have this event on the timeline. If you investigate an issue using logs/metrics, it's good to see changes made by the system.

Reply:
Hmm that seems like something that could apply to anything the CP does, why is this creation of the default resources so important that it deserves so many log lines on Info? 

```
2023-03-07T12:37:52.445Z	INFO	controllers.mesh-defaults	ensuring that default mesh resources exist	{"mesh": "default"}
2023-03-07T12:37:52.445Z	INFO	defaults.mesh	ensuring default resources for Mesh exist	{"mesh": "default"}
2023-03-07T12:37:52.546Z	INFO	defaults.mesh	default Retry already exists	{"mesh": "default", "name": "retry-all-default"}
2023-03-07T12:37:52.646Z	INFO	defaults.mesh	default TrafficPermission already exists	{"mesh": "default", "name": "allow-all-default"}
2023-03-07T12:37:52.747Z	INFO	defaults.mesh	default TrafficRoute already exists	{"mesh": "default", "name": "route-all-default"}
2023-03-07T12:37:52.848Z	INFO	defaults.mesh	default Timeout already exists	{"mesh": "default", "name": "timeout-all-default"}
2023-03-07T12:37:52.949Z	INFO	defaults.mesh	default CircuitBreaker already exists	{"mesh": "default", "name": "circuit-breaker-all-default"}
2023-03-07T12:37:52.949Z	INFO	defaults.mesh	Dataplane Token Signing Key already exists	{"mesh": "default"}
2023-03-07T12:37:52.949Z	INFO	controllers.mesh-defaults	marking mesh that default resources were generated	{"mesh": "default"}
```

All resources get a line even though they have a creation timestamp. We even have an earlier message "ensuring that default..." right before this. It seems like one line plus the presence or absence of the annotation/resources would then be more than enough to debug an issue. But this PR isn't about log messages ultimately so I'll leave them alone

Reply:
> Hmm that seems like something that could apply to anything the CP does

If that's an update operation on the object then yes.

We modify Mesh object to mark this event, I feel like this is somewhat useful in the end.